doc/architecture/decisions/0009-primary-person.md

```diff
@@ -0,0 +1,58 @@
+# 9. Primary Person
+
+## Context
+
+We need to be able to identify/uniquely handle responses and the routing path of the primary person within a household. Currently people within a household would be represented as a list, which generates a number of pages distinct from the main routing path.
```

Comment:
I think we need to make it clear there is a separation between the collection of the primary person and the places where we make use of that fact. We handle responses for the primary person in the same way as any other person, the same for the routing path.